webapp/channels/src/components/widgets/loading/loading_wrapper.test.tsx

```diff
@@ -43,15 +43,15 @@ describe('components/widgets/loading/LoadingWrapper', () => {
     ];
     for (const testCase of testCases) {
         test(testCase.name, () => {
-            const wrapper = shallow(
+            const wrapper = mount(
                 <LoadingWrapper
                     loading={testCase.loading}
                     text={testCase.text}
                 >
                     {testCase.children}
                 </LoadingWrapper>,
             );
-            expect(wrapper).toMatchInlineSnapshot(testCase.snapshot);
+            expect(wrapper).toMatchSnapshot(testCase.snapshot);
```

Comment:
Not really opposed but why these changes?

That being said, if we do it like this, we can remove the `testCase.snapshot` from here, and all the snapshot in the test cases in the file.

Reply:
yes we can do that too. I changed from `toMatchInlineSnapshot` to  `toMatchSnapshot` because when the data passed was empty then the functional component returned `null` whereas  snapshot expected `"''"`. and this was giving error.

Please suggest

Reply:
Looks good to me. Remove the inline snapshots from the testcase (since they are not used anymore) and that will be all :smile:

Reply:
updated the PR with the required changes. Please have a look.